test/vstest.ProgrammerTests/Fakes/FakeEnvironment.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.VisualStudio.TestPlatform.PlatformAbstractions;
+using Microsoft.VisualStudio.TestPlatform.PlatformAbstractions.Interfaces;
+
+namespace vstest.ProgrammerTests.Fakes;
+
+internal class FakeEnvironment : IEnvironment
```

Comment:
Shouldn't we take something as argument to this type? Because currently it's just a proxy to `PlatformEnvironment` so there is no way to mock any behavior or anything.

Reply:
the idea is to prepare the Fake but I don't know what will be the usage we want to have for this fake, but at least is ready to be updated, @nohwnd wdyt? Do you want a full implementation now? If so how do you want it?Callbacks?

Reply:
I am fine if we don't do the implementation but let's add a todo comment. For the implementation, I would expect (but let's wait for @nohwnd feedback) to have all properties being get/set so we can change the behavior to whatever we want and for the "get" methods to have a complementary set method (or if we don't plan to change just take values as ctor param). Actually these fakes could be good fit for record.

Reply:
the problem is that at the moment I don't have any idea of the correct values to avoid to break the tests and I would not spend hours to review all the fakes works right now but have this one merged asap.
If @nohwnd gives me correct default value I can create a correct fake.

Reply:
@Evangelink I had offline sync with Jakub and he's ok with the current implementation for now, we'll amend it if needed in future.